.travis.yml

```diff
@@ -1,20 +1,15 @@
-sudo: false
+sudo: required
```

Comment:
and why do we require sudo now?

Reply:
This enables the VM instead of a container: https://docs.travis-ci.com/user/reference/overview/.